infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/InfinispanRemoteProperties.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote;
+
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.ogm.datastore.keyvalue.cfg.KeyValueStoreProperties;
+
+/**
+ * Properties for configuring the Infinispan Remote datastore via {@code persistence.xml} or
+ * {@link StandardServiceRegistryBuilder}.
+ */
+public final class InfinispanRemoteProperties implements KeyValueStoreProperties {
+
+	/**
+	 * The configuration property to use as key to define a custom configuration resource
+	 * for the Hot Rod (Infinispan remote) client.
+	 */
+	public static final String CONFIGURATION_RESOURCE_NAME = "hibernate.ogm.infinispan_remote.configuration_resource_name";
```

Comment:
Does that means the generic host / port won't work and someone must provide some xml configuration file? Or is that optional? It might be good to guide users in the JavaDoc or maybe better doc as to what to prefer.